internal/pkg/cache/add.go

```diff
@@ -153,15 +176,21 @@ func (c *Cache) cloneRemoteGitRegistry(opts *AddOpts) (err error) {
 	if opts.PackName != "" {
 		clonePath = path.Join(clonePath, "packs", opts.PackName)
 	}
-	err = gg.Get(clonePath, fmt.Sprintf("git::%s", url))
+	err := gg.Get(clonePath, fmt.Sprintf("git::%s", url))
 	if err != nil {
 		logger.ErrorWithContext(err, "could not install registry", c.ErrorContext.GetAll()...)
-		return
+		return "n/a", err
+	}
```

Comment:
We could shorten this slightly, but again not required and personal style. Do we need to document how to handle "n/a" and use a const, or is this thinking too much and not required?
```
	if err := gg.Get(clonePath, fmt.Sprintf("git::%s", url)); err != nil {
		logger.ErrorWithContext(err, "could not install registry", c.ErrorContext.GetAll()...)
		return "n/a", err
	}
```